README.md

```diff
@@ -103,6 +103,7 @@ This file contains a number of front-end interview questions that can be used wh
 * Have you played around with the new CSS Flexbox or Grid specs?
 * How is responsive design different from adaptive design?
 * Have you ever worked with retina graphics? If so, when and what techniques did you use?
+* Which one is better translate() vs Position:Abs;Top/left?
```

Comment:
This question is about performance of website. As an front end dev, should know which one of the method is better when moving elements and especially for creating animations using css3.

Here is the detailed explanation on this topic by Paul Irish http://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/